Sources/PointFree/PageLayout.swift

```diff
@@ -9,49 +9,88 @@ import Prelude
 import Styleguide
 import Tuple
 
-func respond<A>(_ view: View<A>, layout: @escaping (Flash?, View<A>) -> View<A>)
+struct SimplePageLayoutData<A> {
+  private(set) var currentRoute: Route?
+  private(set) var currentUser: Database.User?
+  private(set) var extraStyles: Stylesheet
+  private(set) var data: A
+  private(set) var flash: Flash?
+  private(set) var showTopNav: Bool
+  private(set) var title: String
+  private(set) var useHighlightJs: Bool
+
+  init(
+    currentRoute: Route? = nil,
```

Comment:
I provide default values for the layout fields that don't always need to be specified. however, this one in particular _should_ be required, as it's makes the nav properly redirect after a login. but we weren't doing a very good job of threading that info to the nav, so rather than breaking a bunch of stuff and fixing I made it optional

Reply:
Sounds good! We can revisit fixing soon tho.